### PR TITLE
[FIX] point_of_sale: Fix delay barcode scanner

### PR DESCRIPTION
**Pull request by qle-odoo**
When we make a request to /hw_proxy/sanner there are a timeout of 7,5 sec
After this timeout we wait for 5 sec before send another request.
And we clean the queue of barcode in IoT Box after 5 sec.

So if a cashier scan a product between this "gap" the product is not added to the POS

With this commit we check if barcode is connected to a IoT Box and not a Posbox
And if true we remove the delay before send another request.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/point_of_sale/static/src/js/barcode_reader.js [hunk 1]
```diff
@@ -101,6 +101,17 @@ var BarcodeReader = core.Class.extend({
         }
         this.remote_active = 1;
 
+        $.ajax({
```
Comment (pimodoo) [inline addons/point_of_sale/static/src/js/barcode_reader.js:104]: maybe add "this.iotbox = false" in the init to be sure.

Comment (qle-odoo) [reply addons/point_of_sale/static/src/js/barcode_reader.js:104]: yes.. ok

**Comment by pimodoo** [comment]
@robodoo r+ rebase-ff

**Comment by pimodoo** [comment]
@fw-bot up to saas-13.4

**Pull request closed without merging**